lib/test_helper.js

```diff
@@ -6,11 +6,10 @@ const sinon = require( "sinon" ); // eslint-disable-line import/no-extraneous-de
 const pgClient = require( "./pg_client" );
 const esClient = require( "./es_client" );
 
-const environment = process.env.NODE_ENV || global.config.environment;
```

Comment:
I wonder if places like this where we're checking for the test env, we should still consider both `process.env.NODE_ENV` and `global.config.environment`. It's possible to override environment in config.js to some fixed value (not advised or default, but still possible), which would cause `global.config.environment` to be that fixed value and not "test" when running tests. I think it's safer to look at both places. Maybe something like `process.env.NODE_ENV === "test" || global.config.environment === "test"` to be extra cautious.